tests/vm_test.go

```diff
@@ -933,7 +933,7 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 				})
 			})
 
-			Context("Using RunStrategyRerunOnFailure", func() {
+			Context("Using RerunOnFailure run strategy", func() {
```

Comment:
I think there regex will still match "Failure" or not?

Reply:
I don't think `Failure` is enough, it has to be `Failure [`. (I forgot where the regex is defined)

Reply:
Here: https://github.com/kubevirt/project-infra/blob/main/github/ci/prow-deploy/kustom/base/configs/current/config/config.yaml#L37-L46

Reply:
Ok, so can we keep the previous and add `,` or such?

Reply:
We can do anything we want, as long as we break the string "Failure ["